src/UrbanPlan.java

```diff
@@ -556,4 +560,401 @@ public void printStats() {
+							babyOneCom++;
+							childOne[i][j] = boardOne[i][j];
+						}
+						else {
+							childOne[i][j] = this.board[i][j];
+						}
+					}
+					else {
+						childOne[i][j] = boardOne[i][j];
+					}
+
+					//for baby two
+					if(boardTwo[i][j].equals("I")) {
+						if(babyTwoInd < this.numIndustrial) {
+							babyTwoInd++;
+							childTwo[i][j] = boardTwo[i][j];
+						}
+						else {
+							childTwo[i][j] = this.board[i][j];
+						}
+					}
+					else if(boardTwo[i][j].equals("R")) {
+						if(babyTwoRes < this.numResidential) {
+							babyTwoRes++;
+							childTwo[i][j] = boardTwo[i][j];
+						}
+						else {
+							childTwo[i][j] = this.board[i][j];
+						}
+					}
+					else if(boardTwo[i][j].equals("C")) {
+						if(babyTwoCom < this.numCommercial) {
+							babyTwoCom++;
+							childTwo[i][j] = boardTwo[i][j];
+						}
+						else {
+							childTwo[i][j] = this.board[i][j];
+						}
+					}
+					else {
+						childTwo[i][j] = boardTwo[i][j];
+					}
+				}
+				else { //right half
+
+					//for baby one
+					if(boardTwo[i][j].equals("I")) {
+						if(babyOneInd < this.numIndustrial) {
+							babyOneInd++;
+							childOne[i][j] = boardTwo[i][j];
+						}
+						else {
+							childOne[i][j] = this.board[i][j];
+						}
+					}
+					else if(boardTwo[i][j].equals("R")) {
+						if(babyOneRes < this.numResidential) {
+							babyOneRes++;
+							childOne[i][j] = boardTwo[i][j];
+						}
+						else {
+							childOne[i][j] = this.board[i][j];
+						}
+					}
+					else if(boardTwo[i][j].equals("C")) {
+						if(babyOneCom < this.numCommercial) {
+							babyOneCom++;
+							childOne[i][j] = boardTwo[i][j];
+						}
+						else {
+							childOne[i][j] = this.board[i][j];
+						}
+					}
+					else {
+						childOne[i][j] = boardTwo[i][j];
+					}
+
+					//for baby two
+					if(boardOne[i][j].equals("I")) {
+						if(babyTwoInd < this.numIndustrial) {
+							babyTwoInd++;
+							childTwo[i][j] = boardOne[i][j];
+						}
+						else {
+							childTwo[i][j] = this.board[i][j];
+						}
+					}
+					else if(boardOne[i][j].equals("R")) {
+						if(babyTwoRes < this.numResidential) {
+							babyTwoRes++;
+							childTwo[i][j] = boardOne[i][j];
+						}
+						else {
+							childTwo[i][j] = this.board[i][j];
+						}
+					}
+					else if(boardOne[i][j].equals("C")) {
+						if(babyTwoCom < this.numCommercial) {
+							babyTwoCom++;
+							childTwo[i][j] = boardOne[i][j];
+						}
+						else {
+							childTwo[i][j] = this.board[i][j];
+						}
+					}
+					else {
+						childTwo[i][j] = boardOne[i][j];
+					}
+				}
+			}
+		}
+		childOne = mutate(childOne, babyOneInd, babyOneRes, babyOneCom);
+		childTwo = mutate(childTwo, babyTwoInd, babyTwoRes, babyTwoCom);
+
+		children.add(childOne);
+		children.add(childTwo);
+		return children;
+	}
+
+	/**
+	 * Kills lowest board
+	 * Returns original list without the lowest
+	 */
+	public ArrayList<String[][]> kill(ArrayList<String[][]> bigList) {
```

Comment:
I don't think we use this method anymore so you can probably kill the kill method >:D

Reply:
Removed in 9557989